packages/components/src/StatusIndicator/StatusIndicator.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styles from "./StatusIndicator.css";
+
+export type StatusType =
+  | "success"
+  | "warning"
+  | "critical"
+  | "inactive"
+  | "informative";
+
+interface StatusIndicatorProps {
+  readonly status: StatusType;
+}
+
+export function StatusIndicator({ status }: StatusIndicatorProps) {
+  return (
+    <div
+      style={{ backgroundColor: `var(--color-${status}` }}
+      className={styles.statusIndicator}
```

Comment:
Should this be a `<span />` or is it intended to always be `display: block`?

Reply:
could be a span tho i'd add `display: inline-block` on it tho so it properly flows with other inline elements